cmake/urdfdom-config.cmake.in

```diff
@@ -7,17 +7,26 @@ set(@PKG_NAME@_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/include" "@TinyXML_INCLUDE_D
 
 foreach(lib @PKG_LIBRARIES@)
   set(onelib "${lib}-NOTFOUND")
+  set(onelibd "${lib}-NOTFOUND")
   find_library(onelib ${lib}
       PATHS "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@"
     NO_DEFAULT_PATH
     )
-  if(NOT onelib)
+  find_library(onelibd ${lib}@CMAKE_DEBUG_POSTFIX@
+      PATHS "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@"
+    NO_DEFAULT_PATH
```

Comment:
The indentation seems to be off (same as in the untouched `find_library` call above).